root/components/common-macros.tt

```diff
@@ -834,9 +834,9 @@ END -%]
     };
     IF medium.name != '';
         args.medium_name = isolate_text(medium.name);
-        l('{medium_format} {position}: {medium_name}', args);
+        l('<span>{medium_format} {position}:</span> {medium_name}', args);
```

Comment:
Right, though I think it's good to avoid CSS selectors that depend too much on the structure of the HTML. Unless the style would break if it wasn't a direct child of `<a>`.

About ambiguity, I mean if we change the surrounding HTML later and the generic `<span>` no longer works, we'll have to change the string again (unless it has a class name we could use).

Feel free to change it to whatever you think is best (or we can merge it as-is).